twitter_delete.rb

```diff
@@ -96,22 +98,33 @@ def api_call(method, *args)
   tweets_to_delete += tweets.reject(&method(:too_new_or_popular?))
 end
 
+puts "==> Checking timeline to API Completed"
+
 if @options[:archive_given]
-  puts "==> Checking archive JS..."
+  puts "==> Checking archive JS part = ".concat(@options[:part].to_s)
+  
   archive_tweet_ids = []
 
   # tweet.js is not valid JSON...
+  puts "==> Fixing tweet.js is not valid JSON"
   file_contents = File.read(@options[:archive])
-  file_contents.sub! "window.YTD.tweet.part0 = ", ""
+  file_contents.sub! "window.YTD.tweet.part".concat(@options[:part].to_s).concat(" = "), ""
 
-  JSON.parse(file_contents).each do |tweet|
-    archive_tweet_ids << tweet["id_str"]
-  end
+  puts "==> Parsing tweet.js"
+  tweets = JSON.parse(file_contents)
 
-  archive_tweet_ids.each_slice(MAX_TWEETS_PER_REQUEST) do |tweet_ids|
+  puts "==> Found #{tweets.size} tweets in archive data"
+  
+  tweets.each do |tweet|    
+    archive_tweet_ids << tweet["tweet"]["id_str"]
+  end  
+
+  puts "==> Checking archive to API..."
```

Comment:
Can you explain this? It doesn't make sense as-is.

Reply:
After fix not valid JSON and parsed json. I'm giving it for report status checking archive from twitter API.

Reply:
How about:

```suggestion
  puts "==> Querying archived tweets from API..."
```